File: mario-more/mario.c
//---------------------------------
//Markus Dwiyanto Tobi Sogen
//CS50 for teachers
//Indonesia
//mario-more.c
//---------------------------------
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");  //Input height first
    }

    while
    ((height < 1) || (height > 8));  // boolean logic use OR

    // for loop
    for (int row = 0; row < height; row++)

    {

        for (int column = row + 1; column < height; column++)
        {
            printf(" "); //space
        }


        for (int pagar = height + row + 1; pagar > height; pagar--) //right
        {
            printf("#");
        }
        printf("  ");

        for (int pagar = height + row + 1; pagar > height; pagar--) //left
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# ---------------------------------
# Markus Dwiyanto Tobi Sogen
# CS50 for teachers
# Indonesia
# mario-more.py
# ---------------------------------
# TODO
from cs50 import get_int
# cek if user input number true or false
while True:
    height = get_int("Height:  ")
    # try check number input from user
    if height > 0 and height < 9:
        break

for i in range(height):
        # Print the left side
        for j in range(height - i - 1):
            print(" ", end="")
        for j in range(i + 1):
            print("#", end="")
        # Print space
        print("  ", end="")
        # Print the right side
        for j in range(i + 1):
            print("#", end="")
        print()